sky/tools/create_full_ios_framework.py

```diff
@@ -192,12 +233,19 @@ def zip_archive(dst):
       'Flutter.xcframework',
       'entitlements.txt',
       'without_entitlements.txt',
+      'extension_safe/Flutter.framework',
+      'extension_safe/Flutter.xcframework',
+      'extension_safe/sim',
   ],
                         cwd=dst)
   if os.path.exists(os.path.join(dst, 'Flutter.dSYM')):
```

Comment:
We explictly put the Flutter.dsym in a standalone zip file. So I thought we should do the same for extension safe dsym to keep it consistent

Reply:
Oh I think this is what is misleading. I don't think the zip file location mattered, they are not really useful here, they are later uploaded to a destination determined in the mac_ios_engine.json. After my change, both Flutter.dsym and extension_safe_Flutter.dsym are uploaded to ios/release, where Flutter.dsym used to be.